### ISSUE TITLE
master build is broken

### ISSUE THREAD

**Issue by fbs** [iovisor/bpftrace#2157]
Seems like teh llvm+btf builds fail on the tools split

**Comment by fbs** [comment]
```
3: BTF: using data from /sys/kernel/btf/vmlinux
3: BTF: using data from /sys/kernel/btf/vmlinux
3: definitions.h:6:10: fatal error: 'sys/socket.h' file not found
3: ERROR: Input (147): #include <__btf_generated_header.h>
3: #ifndef BPFTRACE_HAVE_BTF
3: #include <linux/socket.h>
3: #include <net/sock.h>
3: #else
3: #include <sys/socket.h>
3: #endif
```

**Comment by viktormalik** [comment]
This is related to #2150. Fix is in #2149, I'm just waiting for review there.

**Comment by viktormalik** [comment]
Fixed by #2149.
